test/command/suite/select/filter/ordered_near_phrase/max_element_intervals/ngram.expected

```diff
@@ -0,0 +1,48 @@
+table_create Entries TABLE_NO_KEY
+[[0,0.0,0.0],true]
+column_create Entries content COLUMN_SCALAR Text
+[[0,0.0,0.0],true]
+table_create Terms TABLE_PAT_KEY ShortText   --default_tokenizer 'TokenNgram("unify_alphabet", false,                                   "unify_digit", false)'   --normalizer NormalizerNFKC121
+[[0,0.0,0.0],true]
+column_create Terms entries_content COLUMN_INDEX|WITH_POSITION Entries content
+[[0,0.0,0.0],true]
+load --table Entries
+[
+{"content": "abc1def"},
+{"content": "abc12def"},
+{"content": "abc123def"}
+]
+[[0,0.0,0.0],3]
+select Entries --filter 'content *ONP-1,0,5 "abc def"' --output_columns '_score, content'
```

Comment:
Could you also add a separated test for `n_phrases > n_max_element_intervals` case?

Reply:
This case satisfies `n_phrases > n_max_element_intervals` because `n_phrases` is 2 (`abc` and `def`), `n_max_element_intervals` is `1` (`5` of `*ONP-1,0,5`).

Did you mean `n_phrases <= n_max_element_intervals` case?

Reply:
Hmm, I feel like `n_phrases > n_max_element_intervals + 1` is correct in implementation.

Reply:
Added tests for `n_phrases > n_max_element_intervals + 1` and `n_phrases == n_max_element_intervals + 1` and `n_phrases < n_max_element_intervals + 1`.